packages/browser/src/feedbackAsync.ts

```diff
@@ -0,0 +1,7 @@
+import { buildFeedbackIntegration } from '@sentry-internal/feedback';
+import { lazyLoadIntegration } from './utils/lazyLoadIntegration';
+
+// This is for users who want to have a lazy-loaded feedback widget
+export const feedbackAsyncIntegration = buildFeedbackIntegration({
+  lazyLoadIntegration,
```

Comment:
This whole `buildFeedbackIntegration` exists, and is called from inside the browser package, so that we can inject `lazyLoadIntegration` instead of creating a circular dependency between `@sentry/browser` and `@sentry/feedback`.